lib/remote/httpserverconnection.cpp

```diff
@@ -537,8 +533,6 @@ void HttpServerConnection::ProcessMessages(boost::asio::yield_context yc)
 			auto authenticatedUser (m_ApiUser);
 
 			if (!authenticatedUser) {
-				CpuBoundWork fetchingAuthenticatedUser (yc);
-
```

Comment:
Well, the most problematic ones here are ApiUser::GetByName() and Utility::ComparePasswords(), which is slow on purpose btw..

Reply:
I don't see any problems with the methods you mentioned, neither of these acquire any mutex, so why do you want to yield that coroutine just because you're going to use `ApiUser::GetByName()` and `Utility::ComparePasswords()`? This is a waste of time and semaphores.

Reply:
Oh! `ConfigType::GetObject()` does lock a mutex, but still not that many threads are performing `ApiUser::GetByName()` simultaneously.